src/components/EditCoursePage/EditCourseForm.jsx

```diff
@@ -281,6 +297,36 @@ export class BaseEditCourseForm extends React.Component {
               disabled={disabled}
               required={isSubmittingForReview}
             />
+            <FieldLabel
+              id="collaborators.label"
```

Comment:
Is this meant to be the string ```"collaborators.label"``` or the value of the ```collaborators``` object's ```label``` key?

Reply:
the string. this is inkeeping with the rest of publisher.